cliquet/tests/testappviews.py

```diff
@@ -0,0 +1,11 @@
+from cliquet.resource import BaseResource, ResourceSchema, crud
```

Comment:
I believe we should put that in a separate folder, exactly like we would do for a real app.